pkg/azureclients/diskclient/interface.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	// APIVersion is the API version for compute.
-	APIVersion = "2020-06-30"
+	APIVersion = "2020-12-01"
```

Comment:
is it possible to set a new APIVersion for edge, this would break on some regions since `2020-12-01
` is quite new, could set similar to `AzureStackCloud` in the below

Reply:
API is available in all clouds except USNAT. Hopefully that will be available sometime soon.

Reply:
that could still be an issue. What's the cloud name for edge cloud, is it possible to only set this api-version for edge? thanks.

Reply:
@MirzaSikander could you confirm when would USNAT support this API version?

Reply:
@MirzaSikander any updates on this?